src/lib/services/feature-toggle-service.ts

```diff
@@ -165,7 +165,7 @@ class FeatureToggleService {
         projectId,
     }: IFeatureContext): Promise<void> {
         const id = await this.featureToggleStore.getProjectId(featureName);
-        if (id !== projectId) {
+        if (id && id !== projectId) {
```

Comment:
Don't fail if we don't find a project id associated with the feature, instead, let the application return not found.